prow/plugins/cla/cla_test.go

```diff
@@ -332,6 +373,42 @@ func TestCheckCLA(t *testing.T) {
 
 			removedLabel: fmt.Sprintf("/#3:%s", labels.ClaYes),
 		},
+		{
+			name: "cla/linuxfoundation status retains the cla-no label and removes cla-yes label when its state is \"failure\"",
+			contexts: map[string]string{
+				"cla/linuxfoundation": "failure",
+				"EasyCLA":             "success",
+			},
```

Comment:
Thanks for the refactor down here.  I think I'm asking for a similar refactor to the status event handling test

Reply:
sure. will do :)